x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/job_type.tsx

```diff
@@ -22,23 +22,23 @@ export const JobType: FC<Props> = ({ type, setFormState }) => {
     'xpack.ml.dataframe.analytics.create.outlierDetectionHelpText',
     {
       defaultMessage:
-        'Outlier detection jobs require a source index that is mapped as a table-like data structure and analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
+        'Outlier detection identifies unusual data points in the dataset. Outlier detection jobs analyze only numeric and boolean fields. Use the advanced editor to add custom options to the configuration.',
```

Comment:
Since I think this message is associated with the "job type" field, would it be possible to limit it to just this sentence:

> Outlier detection identifies unusual data points in the data set.
(Note "data set" is two words in our style guide).

> Outlier detection jobs analyze only numeric and boolean fields.
This seems like it's more applicable when you're looking at the "included fields" section. (Assuming that non-numeric/boolean fields are automatically excluded?)

> Use the advanced editor to add custom options to the configuration.
This seems to be pertinent to the "switch to json editor" or maybe the subsequent "advanced" stage of the wizard.  I think it's more useful there than here with the job type.

Reply:
Updated job text in f3feaac3a3bbf55688e365081e904507aef754bc

Reply:
Is it worth adding the `Use the advanced editor to add custom options to the configuration.` to here:

![image](https://user-images.githubusercontent.com/7405507/87689701-6e63d880-c780-11ea-82f3-a0a4f3c7a514.png)